src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -140,6 +140,12 @@ public interface IMethodSymbol : ISymbol
         /// </summary>
         bool IsReadOnly { get; }
 
+        /// <summary>
+        /// 'init' set accessors can only be invoked during construction or in the initialization phase
```

Comment:
>set [](start = 19, length = 3)

It isn't clear if this is intended to apply only to set accessors, or if other methods may be so flagged.